js/src/Ice/RequestHandlerCache.js

```diff
@@ -85,7 +85,15 @@ export class RequestHandlerCache {
                 ex instanceof CloseConnectionException ||
                 ex instanceof ObjectNotExistException)
         ) {
-            return RequestHandlerCache.checkRetryAfterException(ex, this._reference, cnt);
+            try {
+                return RequestHandlerCache.checkRetryAfterException(ex, this._reference, cnt);
+            } catch (e) {
```

Comment:
You can't catch a specific type in JS?

Reply:
No, there is just `catch (e)` without type.